src/assets/pleasew

```diff
@@ -0,0 +1 @@
+dummy
```

Comment:
To make sure I understand... these are the files you'd get with 'normal' go embed, but when you build with plz it sets up the real ones instead?

Reply:
Yup!